src/Service.Tests/Authorization/AuthorizationHelpers.cs

```diff
@@ -53,8 +54,10 @@ public static AuthorizationResolver InitAuthorizationResolver(RuntimeConfig runt
 
             metadataProvider.Setup(x => x.GetEntityName(It.IsAny<string>()))
                 .Returns((string entity) => entity);
+            Mock<IMetadataProviderFactory> metadataProviderFactory = new();
+            metadataProviderFactory.Setup(x => x.GetMetadataProvider(It.IsAny<string>())).Returns(metadataProvider.Object);
```

Comment:
if this is still remaining a single database test, can `x.GetMetadataProvider(It.IsAny<string>()))` be `x.GetMetadataProvider(string.Empty))`?

Reply:
fixed this to use the actual datasourceName.